packages/rn-tester/js/examples/TextInput/TextInputSharedExamples.js

```diff
@@ -815,14 +835,9 @@ function MultilineStyledTextInput({
 
 module.exports = ([
   {
-    title: 'Auto-focus',
+    title: 'Auto-focus & select text on focus',
```

Comment:
Why haven't you just added `selectTextOnFocus` here but instead you also added `onChangeText`?

Reply:
To show selected text I have added `value` prop with initial value set to `Hello World`. Earlier the `auto-focus` sample allowed user input. After adding `value` prop user input was getting overridden, I have added `onChangeText` to change value to user's input.

Reply:
@cortinico do you want me to remove onChangeText?

Reply:
nope, thanks for the clarification 👍